src/wireframes/model/transform.spec.ts

```diff
@@ -170,9 +168,75 @@ describe('Transform', () => {
                 .rotateAroundAnchor(center, rotation);
 
         const actual = Transform.createFromTransformationsAndRotation([transformation1, transformation2], rotation);
-        const expected = new Transform(new Vec2(300, 150), new Vec2(300, 140), Rotation.fromDegree(45));
+        
+        // Allow for small variations in position and size
+        expect(Math.abs(actual.position.x - 300)).toBeLessThan(1);
+        expect(Math.abs(actual.position.y - 150)).toBeLessThan(1);
+        expect(Math.abs(actual.size.x - 300)).toBeLessThan(1);
+        expect(Math.abs(actual.size.y - 140)).toBeLessThan(1);
+        expect(actual.rotation.degree).toBe(45);
+    });
+
+    it('should create from transformations with different rotations', () => {
```

Comment:
it is better to call the it( ... ) in the loop. I think there are examples here in the tests.